package.json

```diff
@@ -27,5 +27,8 @@
     "html-minifier-terser": "^7.2.0",
     "inliner": "^1.13.1",
     "nodemon": "^3.1.7"
+  },
+  "engines": {
+    "node": ">=21.0.0"
```

Comment:
Should this not be 20 then not 21?

Reply:
I was working with different folders and got a bit confused. I changed it to 20.